framework/src/main/java/org/checkerframework/framework/type/poly/AbstractQualifierPolymorphism.java

```diff
@@ -69,6 +70,9 @@
     /** Replaces each polymorphic qualifier with its instantiation. */
     private AnnotatedTypeScanner<Void, AnnotationMirrorMap<AnnotationMirrorSet>> replacer;
 
+    /** Utility class for working with {@link TypeMirror}s. */
+    private final Types typeUtils;
```

Comment:
I intentionally used `typeUtils` here instead of the common `types`, because there already was at least one use of a `types` local variable or parameter. Shadowing the name would have worked, but in general, I think `typeUtils` is the better name for the variable - it's also the name of the getter from `ProcessingEnvironment.getTypeUtils`.
For many existing uses of `types` it would be a breaking change to rename to `typeUtils`, so when we next have breaking changes we can also perform that renaming.